### DISCUSSION TITLE
Wireguard Setup?

### DISCUSSION THREAD

**Discussion by NylonDiamond** [Jip-Hop/jailmaker#146] [Q&A]
I want to set up wireguard in a jail. I'd prefer to use `wg-easy` (a docker image making wireguard setup fast and easy) in my docker jail, if possible. 
I found this discussion about setting up wireguard  in a jail: https://github.com/Jip-Hop/jailmaker/issues/98. 
I set up a bridge called `br0` in truenas scale (_truenas IP: 172.168.100.1_) and used that bridge in my docker jail. 
I added this to my docker jail config:
`
systemd_nspawn_user_args=--network-bridge=br0
                                              --resolv-conf=bind-host
`

I also port forwarded 51820 for the docker jail's IP on my router. (_jail IP: 172.168.100.2_)

I have a few containers running in that jail and they all work fine and connect to the internet.
But I can never connect the the wg-easy one from my phone when I activate the wiregurad VPN connection. 
Has anyone had any luck running wireguard inside a jail with docker? I don't see any errors in the logs but I can never make a wireguard connection. 

Thanks!

**Comment by Jip-Hop** [comment]
First try to connect to your wireguard container from inside your LAN. If that works make the setup more complicated by connecting via WAN and port forwarding. If you have an OpenWrt capable router you could run wireguard on there instead. Other than that I have no advice.

**Reply by NylonDiamond** [reply]
I'll give it another shot. @joeroback if you have any advice, let me know. Thanks

**Comment by joeroback** [comment]
yea my wireguard setup is wireguard -> protonVPN and i want all traffic to go through the VPN, which is the opposite of wg-easy.
